workers/handshake-manager/src/manager.rs

```diff
@@ -774,7 +774,8 @@ impl HandshakeExecutor {
             self.global_state.clone(),
```

Comment:
getting a clippy error here saying the `self.global_state` is still the old `RelayerState` type instead of the new `State` type

Reply:
handshake manager isn't refactored yet, errors here are expected. I'm fixing the tasks first then the callers of tasks.